hyperspace/solana/src/instructions.rs

```diff
@@ -0,0 +1,41 @@
+use anchor_client::anchor_lang::prelude::*;
+use anchor_client::anchor_lang;
+
+#[derive(Debug, Clone, PartialEq)]
+pub struct Deliver {
+  pub messages: Vec<AnyCheck>,
+}
+impl borsh::ser::BorshSerialize for Deliver
```

Comment:
Isn't there a macro for this?

Reply:
Yes we can use the `AnchorSerialize` and `AnchorDeserialize` macro.